gremlin-go/driver/result.go

```diff
@@ -21,18 +21,206 @@ package gremlingo
 
 // AN-968 Finish Result implementation.
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+	"reflect"
+	"strconv"
+)
 
 // Result Struct to abstract the Result and provide functions to use it.
 type Result struct {
 	result interface{}
 }
 
-// AsString returns the current Result formatted as a string.
-func (r *Result) AsString() string {
+// ToString returns the string representation of the Result struct in Go-syntax format
+func (r *Result) ToString() string {
```

Comment:
`ToString` is kind of a JavaScript-ism to me. `String` would be more idiomatic imo